sourced/ml/algorithms/uast_id_distance.py

```diff
@@ -0,0 +1,134 @@
+from collections import namedtuple
+from typing import Iterable, Tuple, Union
+from itertools import combinations
+
+import bblfsh
+
+from sourced.ml.algorithms.uast_ids_to_bag import UastIds2Bag
+from sourced.ml.utils import bblfsh_roles
+
+
+Point = namedtuple("Point", ["token", "info"])
```

Comment:
A bit weird name... Can you elaborate what it does

Reply:
you measure a distance between points, so anything you need for it inside.

Reply:
Write it in the comment